packages/flutter_test/lib/src/binding.dart

```diff
@@ -848,6 +854,13 @@ class AutomatedTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
     });
 
     return new Future<Null>.microtask(() async {
+      // Run all queued microtasks.
+      await new Future<Null>.microtask(() {});
+      // When the test had an exception, the test-framework already ran the
+      // teardown functions, removing the _fakeAsync object.
+      if (fakeAsync == null)
+        return null;
```

Comment:
Why is this a problem now but not before?

Reply:
Seems like the code around it has changed since I wrote this patch. That part is removed now.